Makefile

```diff
@@ -108,18 +111,30 @@ install-hypershift:
 install-helm:
 	@$(TOOLS_SCRIPT) install-helm
 
+create-day2-cluster:
+	@$(CLUSTER_SCRIPT) create-day2-cluster
+
+get-worker-iso:
```

Comment:
It probably should depend on create-day2-cluster and no point to have create-cluster-iso